lib/elixir_console_web/live/console_live/command_input_component.ex

```diff
@@ -0,0 +1,100 @@
+defmodule ElixirConsoleWeb.ConsoleLive.CommandInputComponent do
+  @moduledoc """
+  Live component module that implements the portion of the application where
+  users prompt the commands. It also handle special key events to mimic the iEX
+  user experience.
+  """
+
+  use Phoenix.LiveComponent
+  alias ElixirConsole.Autocomplete
+
+  def render(assigns) do
+    Phoenix.View.render(ElixirConsoleWeb.ConsoleView, "command_input.html", assigns)
+  end
+
+  def mount(socket) do
+    {:ok,
+     assign(
+       socket,
+       history_counter: 0,
+       input_value: "",
+       caret_position: 0
+     )}
+  end
+
+  # TAB KEY
+  def handle_event("suggest", %{"keyCode" => 9, "value" => value}, socket) do
```

Comment:
Would it make sense to define constants with the keycode values? Eg: `@tab_keycode 9`

Reply:
That would save us the comment on the previous line too I guess